webpack.commons.js

```diff
@@ -71,6 +71,8 @@ module.exports = {
     })
   ],
   devServer: {
+    host: '0.0.0.0',
+    disableHostCheck: true,
```

Comment:
```suggestion
```

Reply:
Why? It was part of the PR.

Reply:
maybe not everyone wants to expose their dev environment to the whole network by default.
other than that is should be in a separate PR.

Reply:
OK